config/features.yml

```diff
@@ -521,9 +521,6 @@ features:
     actor_type: user
     description: enables adding applicable flashes to disability_526 prior to submission.
     enable_in_development: true
-  disability_526_new_confirmation_page:
```

Comment:
Reminder to remove this in rails console in the terminal after merging. Thanks!

Reply:
Done!